app/src/routes.js

```diff
@@ -11,7 +11,14 @@ export default (store) => {
   return [
     {
       path: `/`,
-      redirect: `/cosmos-hub/portfolio`,
+      beforeEnter: async (to, from, next) => {
+        const userSignedIn = await store.dispatch("checkSession")
+        if (userSignedIn) {
+          next("/notifications")
+        } else {
+          next("/cosmos-hub/portfolio")
```

Comment:
For the future, it would be cool to have stored in localStorage the last network an user visited, so it returns to the same, `favoriteNetwork`